src/views/EdgeApplicationsRulesEngine/ListView.vue

```diff
@@ -75,7 +74,16 @@
     }
   })
 
+  const drawerRulesEngineRef = ref('')
   const hasContentToList = ref(true)
+  const listRulesEngineRef = ref(null)
+
+  const phaseOptions = ref(['Request phase', 'Response phase'])
```

Comment:
Não precisa ser reativo pelo que vi.

```suggestion
  const PHASE_OPTIONS = ['Request phase', 'Response phase']
```